packages/sdk/package.json

```diff
@@ -87,7 +87,7 @@
     "typescript": "5.0.3"
   },
   "peerDependencies": {
-    "@soundxyz/sound-protocol": "~1.4.0"
+    "@soundxyz/sound-protocol-private": "1.5.0-alpha-1be6075.1"
```

Comment:
Shouldn't it be major bump on the protocol package side?

Reply:
v1 minters are still in that package, and this only adds new ones, so should be no breaking change, just new code